apix-docker/build.gradle

```diff
@@ -8,16 +8,19 @@ subprojects {
         leanImage = true
     }
 
-    // Isolate the version number ("docker-52" -> "52"). We should find a cleaner way
-    def subproject_alfresco_version = project.name.substring(project.name.length() - 2)
-
+    // Isolate the version number ("docker-231" -> "231"). We should find a cleaner way
+    def subproject_alfresco_version = project.name.substring(project.name.length() - getTotalNumbersAfterDash(project.name))
```

Comment:
This _should_ be replaced by overrides.gradle instead of string manipulation, but it is ok to keep it out of scope for this PR. What I woumd do however is replace this line by:
`def subproject_alfresco_version = project.name.split("-").last() ` and remove the getTotalNumbersAfterDash function.